testapi.pm

```diff
@@ -1937,7 +1937,7 @@ sub data_url($) {
         return autoinst_url("/assets/repo/" . get_var($name));
     }
     if ($name =~ /^ASSET_\d$/) {
```

Comment:
Id anyone aware why we limit number of assets to 10? Is there any good reason not to make it `/^ASSET_\d+$/`?

Reply:
Not sure. At least we have already `/^ASSET_\d+$/` in openQA's `asset_type_from_setting` function. But maybe there's still some other code which is more limited.

Reply:
I guess we can change it with proper testing, of course. Thanks for the input!